qa/common/shm_util.py

```diff
@@ -388,4 +395,9 @@ def __init__(self, prefix='triton_python_backend_shm_region'):
                     triton_shm_monitor.SharedMemoryManager(shm_region))
 
     def Probe(self):
-        return self.ShmLeakProbe(self._shm_monitors)
+        # Jetson cleanup takes too long and results in false positives.
+        # Do not use the shared memory check on Jetson.
```

Comment:
I personally would like to see this get re-enabled at some point as shared memory can be platform dependent. Can we put a `FIXME` and reference a ticket to track this?

Reply:
Good suggestion, added.